src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/X86/AvxVnni.cs

```diff
@@ -0,0 +1,72 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.CompilerServices;
+
+namespace System.Runtime.Intrinsics.X86
+{
+    [Intrinsic]
+    [CLSCompliant(false)]
+    public abstract class AvxVnni : Avx2
```

Comment:
Now that https://github.com/dotnet/runtime/pull/52419 is merged, you'll want to annotate this with the new `[RequiresPreviewFeatures]` attribute (from `System.Runtime.Versioning` namespace).